code/modules/modular_computers/hardware/job_disk.dm

```diff
@@ -30,9 +30,11 @@
 	if(disk_flags & DISK_POWER)
 		progs_to_store += new /datum/computer_file/program/power_monitor(src)
 		progs_to_store += new /datum/computer_file/program/supermatter_monitor(src)
+		progs_to_store += new /datum/computer_file/program/alarm_monitor(src)
 
 	if(disk_flags & DISK_ATMOS)
 		progs_to_store += new /datum/computer_file/program/atmosscan(src)
+		progs_to_store += new /datum/computer_file/program/alarm_monitor(src)
 
```

Comment:
If a disk has both power and atmos like the CE, will it appear twice?

Reply:
![image](https://github.com/BeeStation/BeeStation-Hornet/assets/130587823/b2a2d3cc-f75b-4ee2-9c45-15162f120efd)
Not at all